### ISSUE TITLE
Save Text from Collect Element in an Element

### ISSUE THREAD

**Issue by ms-mialingvo** [asterics/AsTeRICS-Grid#424]
The very basic version of issue  #301: Option to easily copy the text from the collect element and paste it into an empty element.

**Comment by ms-mialingvo** [comment]
As issue #301 will take longer, I'm changing the request to:
"paste" button in the 'general' tab of 'edit grid item', that allows to paste text that has been previously copied from the collect element into the label field.
The button could either in the place of the "search for images" button with than one going below the label field
or
below the label field.

**Comment by klues** [comment]
Since this still needs the user to open the edit view and the edit dialog, what about this:
A new action (probably a type of "collect element action") which pastes the current collect element content to the next empty element of the current grid or to a new element created on the current grid.
What do you think? Probably also quite easy to implement.

**Comment by ms-mialingvo** [comment]
That sounds useful :) I will double-check with some AAC users first, though.

**Comment by loretod** [comment]
Yes, I second this flow. It would helpful for users to collect commonly used phrases that they generate themselves not having to rely on an editor. Or an option of where to send the collected phrases to in the "collect element action". Letting users create a quick chat page.
